docs/architecture/pairing-with-use-integration-system.md

```diff
@@ -0,0 +1,86 @@
+# Pairing Adapter for Integration System
```

Comment:
What do you think about title "Adapter for pairing Application with Integration System"? Or, similarly to the first document "Pairing Application with Integration System"